### ISSUE TITLE
Migration Rollback Error: (Laravel 5.6, NeoEloquent 1.4.6)

### ISSUE THREAD

**Issue by tumainimosha** [Vinelab/NeoEloquent#288]
Hi. I am new to NeoEloquent. I have been able to implement and run migration, however, facing issue when calling rollback. Migrate works fine, but rollback throws below trace below

```
./artisan neo4j:migrate:rollback

   Vinelab\NeoEloquent\QueryException  : ParameterNotFoundException: Expected parameter(s): migration, migration

  at /path-to-project/vendor/vinelab/neoeloquent/src/Connection.php: 467

Exception trace:

1   Vinelab\NeoEloquent\Connection::run("MATCH (neoEloquentMigration:`NeoEloquentMigration`) WHERE neoEloquentMigration.migration = {migration} OPTIONAL MATCH (neoEloquentMigration:`NeoEloquentMigration`)-[r]-()  WHERE neoEloquentMigration.migration = {migration} DELETE  neoEloquentMigration,r", Object(Closure))
      /path-to-project/vendor/vinelab/neoeloquent/src/Connection.php : 222
2   Vinelab\NeoEloquent\Connection::affectingStatement("MATCH (neoEloquentMigration:`NeoEloquentMigration`) WHERE neoEloquentMigration.migration = {migration} OPTIONAL MATCH (neoEloquentMigration:`NeoEloquentMigration`)-[r]-()  WHERE neoEloquentMigration.migration = {migration} DELETE  neoEloquentMigration,r")
      /path-to-project/vendor/laravel/framework/src/Illuminate/Database/Connection.php : 435
```
I have found similar open issue #192 for different version of Laravel (4.2) and NeoEloquent (1.4.2)

Any help would be appreciated.

**Comment by SanjayVQLIVE** [comment]
Hey did you fix this? Please let me know. I am also facing the same problem.

**Comment by holix** [comment]
This is fixed in 1.4.7
